pycycle/elements/flow_start.py

```diff
@@ -41,9 +45,13 @@ def setup(self):
         set_stat_MN = SetStatic(mode="MN", thermo_data=thermo_data,
                                 init_reacts=elements, fl_name="Fl_O:stat")
 
-        self.add_subsystem('exit_static', set_stat_MN, promotes_inputs=('MN', 'W'),
+        self.add_subsystem('exit_static', set_stat_MN, promotes_inputs=('MN', 'W', 'init_prod_amounts'),
                            promotes_outputs=('Fl_O:stat:*', ))
 
+        if WAR > 0:
```

Comment:
Need to move this if condition into the WAR component. Handle it internally in the way you output init_prod_amounts